app/models/payment.rb

```diff
@@ -1,5 +1,5 @@
 class Payment < ApplicationRecord
-  has_many :claims
+  has_many :claims, dependent: :nullify
```

Comment:
Why was this removed?

Reply:
I'm not sure what you mean here? I've had to add `dependent: :nullify` because deleting payments results in an `ActiveRecord::InvalidForeignKey` error, because the Payment ID isn't removed from the claim on destroy.

Reply:
Oh, I mistyped, I meant added. I was wondering why it was added as I was under the impression that the default was `:nullify` for `has_many` relations but it looks like I'm wrong about that!

Reply:
👍